ccd-definition/CaseEvent.json

```
@@ -200,5 +200,18 @@
     "ShowSummary": "N",
     "ShowEventNotes": "N",
     "CallBackURLSubmittedEvent": "http://fpl-service:4000/callback/case-submission"
+  },
+  {
+    "LiveFrom": 42795,
```

Please use date in String format

```suggestion
    "LiveFrom": "01/01/2017",
```

Apply the same in other places where life from date is defined.